sos/report/plugins/rear.py

```diff
@@ -20,12 +20,14 @@ class Rear(Plugin, RedHatPlugin):
     def setup(self):
         # don't collect recovery ISOs or tar archives
         self.add_forbidden_path([
-            '/var/log/rear/*.iso',
-            '/var/log/rear/*.tar.gz'
+            '/var/lib/rear/output/*'
         ])
 
         self.add_copy_spec([
             '/etc/rear/*conf',
+            '/etc/rear/mappings/*',
```

Comment:
I guess no sensitive data can be present here (just to doublecheck if we shouldnt scrub something but imho not).

Reply:
So the mappings directory is a spot to set a custom ip and routes, so yes there will be sensitive data, but it's needed in support if the files are there.

Reply:
IP addresses and such are fine - they'd be obfuscated by `--clean`. But are there any passwords or such that might be set in `/etc/rear/*conf`?

Reply:
It looks like there is options for specifying passwords in the /etc/rear/local.conf file, I've personally never seen them used. But they're a few options with password in the name, in the example conf file /usr/share/rear/conf/default.conf.

Reply:
Oh, my fault there - read the git diff wrong and thought you were adding the `conf` files as well.

Seems we should be good then.